### DISCUSSION TITLE
Is it possible to add OnClick funtion to a node ?

### DISCUSSION THREAD

**Discussion by Rofoy** [networkx/networkx#4779] [Q&A] [answered]
Hi,

I'm creating some graphs with NetworkX and saving them in html files. I would like to navigate through graphs by clicking on nodes.  
I tried to change the title of the node and put a hyperlink instead, but because of this, the title of the edge no longer appears. 

Is there an On Click function on the node that could make me go to the desired html page ?

Thanks !

**Comment by dschult** [comment, marked as answer]
This question depends entirely on how the network is saved to an html file. NetworkX itself does nothing with respect to hyperlinks, etc. You might want to look at the package you are using to draw/save the html files.
